concepts.wip/boolean-logic/.meta/config.json

```
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb for boolean-logic concept",
-  "authors": [],
+  "blurb": "Boolean logic forms part of Boolean algebra, and involves combining logical statemets using logical connectives, which evaluates to either true or false.",
```

Many people find maths terms scary, so perhaps we can remove it from here:

```suggestion
  "blurb": "Boolean logic involves combining logical statements using logical connectives, which evaluate to either true or false.",
```

---

Done. Thanks for the suggestion.